packages/guidecue/src/types.ts

```diff
@@ -0,0 +1,105 @@
+import { Align, Justify } from '@leafygreen-ui/popover';
+import { TooltipProps } from '@leafygreen-ui/tooltip';
+
+// Exclude these from tooltip (tooltip already extends popover props)
+type ModifiedTooltipProps = Omit<
+  TooltipProps,
+  | 'usePortal'
+  | 'justify'
+  | 'align'
+  | 'onClick'
+  | 'trigger'
+  | 'triggerEvent'
+  | 'shouldClose'
+  | 'className'
+  | 'children'
+  | 'onClose'
+>;
+
+interface Standalone {
+  /**
+   * Used to determine which tooltip will be shown. If `numberOfSteps` is `> 1` then the multi-step tooltip will be shown else the stand-alone tooltip will be shown. This number will be displayed in the multi-step tooltip.
+   * @default: 1
+   */
+  numberOfSteps?: 1;
+  /**
+   * Used to display the current step.
+   * @default: 1
+   */
+  currentStep?: number;
+}
+interface Multistep {
+  /**
+   * Used to determine which tooltip will be shown. If `numberOfSteps` is `> 1` then the multi-step tooltip will be shown else the stand-alone tooltip will be shown. This number will be displayed in the multi-step tooltip.
+   * @default: 1
+   */
+  numberOfSteps: number;
```

Comment:
I wonder if we should add text to explain how (or where to look for instructions) to set up a multistep cue

Reply:
I can maybe point to the storybook demo